src/tabbar.cpp

```diff
@@ -192,6 +192,10 @@ void TabBar::closeTab(int index)
     removeTab(index);
     webview->close();
     delete webview;
+
+    // exit full screen on last tab closed
+    if(this->count() <= 2 &&KiwixApp::instance()->getAction(KiwixApp::ToggleFullscreenAction)->isChecked())
```

Comment:
Please keep a space after the if keyword.

Reply:
and after &&

Reply:
done